db/workflow.go

```diff
@@ -74,7 +72,7 @@ func insertInWorkflow(ctx context.Context, db *sql.DB, wf Workflow, tx *sql.Tx)
 	return nil
 }
 
-func insertIntoWfWorkerTable(ctx context.Context, db *sql.DB, wfID uuid.UUID, workerID uuid.UUID, tx *sql.Tx) error {
+func insertIntoWfWorkerTable(wfID uuid.UUID, workerID uuid.UUID, tx *sql.Tx) error {
 	_, err := tx.Exec(`
```

Comment:
We could change the `tx.Exec` calls to `tx.ExecContext` rather than ignoring context.
https://pkg.go.dev/database/sql

Reply:
Thanks for the great idea - done!